src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageRestoreBar.xaml.cs

```diff
@@ -215,10 +216,14 @@ private void OnRestoreLinkClick(object sender, RoutedEventArgs e)
             {
                 if (_projectContextInfo?.ProjectStyle == ProjectModel.ProjectStyle.PackageReference && _isAssetsFileMissing)
                 {
+                    var evt = new RestoreBannerClickedTelemetryEvent(RestoreButtonAction.MissingAssetsFile);
```

Comment:
I remember that we previously show the banner in other places before:

https://blog.ploeh.dk/content/binary/nuget-package-restore-problem.png

Does this telemetry is emitted only for PM UI and for only packageReference projects?

Reply:
No, I will add telemetry for all projects, even though the telemetry needed for Transitive Dependencies is only for PackageReference projects.

The banner on the PMC is also considered for this telemetry.